docs/api/index.md

```diff
@@ -0,0 +1,9 @@
+# Code Documentation
+
+This section provides detailed documentation for all modules, classes, and functions in the Remarx package.
+
+## sentence
+
+Functions for text segmentation and sentence processing, specifically designed for Marx quote identification.
+
+- [sentence](sentence.md): Core functionality for breaking down text into individual sentences with character-level indexing
```

Comment:
This text also seems redundant. Can mkdocs easily generate a table of contents for us? 

We don't want multiple different versions of the module description because they will get out of sync.